pkg/edgeview/Dockerfile

```diff
@@ -4,6 +4,7 @@ ENV PKGS alpine-baselayout musl-utils iproute2 iptables
 RUN eve-alpine-deploy.sh
 
 COPY src/  /edge-view/.
+COPY vendor  /edge-view/vendor
 COPY go.mod /edge-view/.
 COPY go.sum /edge-view/.
 COPY vendor /edge-view/vendor
```

Comment:
Line line is duplicated: `COPY vendor /edge-view/vendor`

Reply:
That way, we can be extra extra sure it gets copied! 😆 

I will fix it. Thanks for seeing that.

Reply:
Fixed